Stripe/StripeiOSTests/STPElementsSessionTest.swift

```diff
@@ -42,6 +42,15 @@ class STPElementsSessionTest: XCTestCase {
         XCTAssertEqual(elementsSession.merchantCountryCode, "US")
         XCTAssertNotNil(elementsSession.paymentMethodSpecs)
         XCTAssertEqual(elementsSession.cardBrandChoice?.eligible, true)
+        XCTAssertTrue(elementsSession.isApplePayEnabled)
         XCTAssertEqual(elementsSession.allResponseFields as NSDictionary, elementsSessionJson as NSDictionary)
     }
+
+    func testDecodedObjectFromAPIResponseMapping_applePayPreferenceDisabled() {
+        var elementsSessionJson = STPTestUtils.jsonNamed("ElementsSession")!
+        elementsSessionJson["apple_pay_preference"] = "disabled"
+        let elementsSession = STPElementsSession.decodedObject(fromAPIResponse: elementsSessionJson)!
+
+        XCTAssertFalse(elementsSession.isApplePayEnabled)
```

Comment:
WDYT about also testing that apple pay is enabled if the key is missing from the json? Like adding:
```
elementsSessionJson.removeValue(forKey: "apple_pay_preference")
XCTAssertTrue(elementsSession.isApplePayEnabled)
```

Reply:
Based on the openapi spec, `apple_pay_preference` is a required field, so I don't think this test is really needed.

Reply:
Ah if it's required, should we put it in https://github.com/stripe/stripe-ios/pull/3054/files#diff-915f8d0cfb04ee0e164b7b46965d24a7417b021815dd585d5362415509c3ffa4R101 ?